README.md

```diff
@@ -134,7 +134,7 @@ If you like this project, give it a ⭐️!
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍